src/go/rpk/pkg/cli/cmd/redpanda/start_test.go

```diff
@@ -122,33 +118,13 @@ func TestParseSeeds(t *testing.T) {
 		},
 		{
 			name:		"it should fail for empty addresses",
-			arg:		[]string{"+1"},
-			expectedErrMsg:	"Couldn't parse seed '+1': empty address",
-		},
-		{
-			name:		"it should fail if one of the addrs is missing an ID",
-			arg:		[]string{"127.0.0.1:1234+0", "domain.com"},
-			expectedErrMsg:	"Couldn't parse seed 'domain.com': Format doesn't conform to <host>[:<port>]+<id>. Missing ID.",
-		},
-		{
-			name:		"it should fail if one of the addrs' ID isn't an int",
-			arg:		[]string{"127.0.0.1:1234+id?", "domain.com+1"},
-			expectedErrMsg:	"Couldn't parse seed '127.0.0.1:1234+id?': ID must be an int.",
+			arg:		[]string{""},
+			expectedErrMsg:	"Couldn't parse seed '': empty address",
 		},
 		{
 			name:		"it should fail if the host is empty",
-			arg:		[]string{" :1234+1234"},
-			expectedErrMsg:	"Couldn't parse seed ' :1234+1234': Empty host in address ' :1234'",
-		},
-		{
-			name:		"it should fail if the port is empty",
-			arg:		[]string{" :+1234"},
-			expectedErrMsg:	"Couldn't parse seed ' :+1234': Empty host in address ' :'",
-		},
-		{
-			name:		"it should fail if the port is empty",
-			arg:		[]string{"host:+1234"},
-			expectedErrMsg:	"Couldn't parse seed 'host:+1234': Port must be an int",
+			arg:		[]string{" :1234"},
```

Comment:
Was there a bug here once, where "empty" (a space `" "`) is different to empty() (`""`)?

Can you add a test?

Reply:
Ignore me, I see it on line 332

Reply:
Maybe it exists on line 332, but I feel the delimiter is different.

Reply:
For sure!

Reply:
Ah, but that test in L332 does indeed catch that case.